.github/workflows/mssql_online.yml

```diff
@@ -0,0 +1,40 @@
+name: MS SQL Server online tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: MS SQL ${{ matrix.mssql }} online test
+    runs-on: [ubuntu-latest]
+    strategy:
+      matrix:
+        mssql: [2019-latest]
+        java: [11]
+
+    steps:
+    - name: Set up JDK ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - uses: actions/checkout@v2
+    - name: Maven repository caching
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: mssql-${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          mssql-${{ runner.os }}-maven-
+    - name: Setup test environment for MS SQL ${{ matrix.mssql }} 
+      run: |
+        ./build/ci/mssql/start-mssql.sh ${{ matrix.mssql }}
+        ./build/ci/mssql/setup-mssql.sh
+    - name: Build GeoTools dependent modules (no tests)
```

Comment:
You could `-Dfmt.skip=true` here too, since that will get checked by the QA tests.

Reply:
yeah, I'll probably end up doing a once-over for mssql/mysql/oracle once merged; so I'll keep that in mind.
It doesn't add much overhead.

Reply:
done in f662cdf